ruby/forwardfinancing.rb

```diff
@@ -90,7 +93,7 @@
 
 # Send an attachment
 
-file_binary = File.open('forwardfinancing.rb', 'rb').read
+file_binary = File.open('./forwardfinancing.rb', 'rb').read
```

Comment:
Adding a `./` to a file path is essentially a [noop](https://en.wikipedia.org/wiki/NOP_(code)). Read more about them: https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths

The only time it should have any meaning is when used as a command path for the shell, to disambiguate the global search path for executable from executable files in the current directory. You don't want someone to sneak a file named `git` into your current working directory, and suddenly gain user level access when you run `git` instead of forcing you to write `./git`.

For anything in your own code dealing with file paths (unless it's something like `system(path)`) this shouldn't be needed.

Reply:
ok